examples/internal/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -2670,12 +2886,27 @@
             "pattern": "publishers/[^/]+/books/[^/]+"
           },
           {
-            "name": "body",
-            "description": "The book to update.\n\nThe book's `name` field is used to identify the book to be updated.\nFormat: publishers/{publisher}/books/{book}",
+            "name": "book",
+            "description": "The book's `name` field is used to identify the book to be updated.\nFormat: publishers/{publisher}/books/{book}",
```

Comment:
Need to preserve the title/summary paragraph here.

Reply:
This is fixed now. Note that the old code had some inconsistency where some places the comment string was used directly, while in another place it would be split on `\n\n` and joined with `. `. I made this consistent to join again with `\n\n`.